crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/AbstractRestCommand.java

```diff
@@ -65,13 +66,13 @@ public AbstractRestCommand() {
      * @return
      * @throws Exception
      */
-    protected Mediator getMediator() {
+    protected Mediator getMediator(RequestMetrics metrics) {
```

Comment:
I suggest not passing RequestMetrics to getMediator like this. It's a bit odd and makes the code a bit messy. See comments in https://github.com/lightblue-platform/lightblue-core/pull/808.